frontend/src/index.tsx

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.hydrate(<App />, document.getElementById('root'));
```

Comment:
Doesn't hydrate only have an effect if you already have server-side rendered HTML?

Reply:
Ah, true will revert